crates/nu-cli/src/commands/date/format.rs

```diff
@@ -24,11 +24,11 @@ impl WholeStreamCommand for Date {
     fn signature(&self) -> Signature {
         Signature::build("date format")
             .required("format", SyntaxShape::String, "strftime format")
-            .switch("raw", "print date without tables", Some('r'))
+            .switch("table", "print date in a table", Some('t'))
```

Comment:
What does the --table switch do?

Reply:
The original `date format` outputs the result in a table by default. I find it bit inconsistent with the new `date now` and `date to-timezone` commands, which print a string now, so I moved it to a switch instead. I personally don't think the switch is very helpful and happy to remove it.

Reply:
So, this reverses the functionality of `raw`? i.e. by default date prints strings and if you want it in a table you pass the `table` parameter. Do I understand this correctly?